extensions/binding/org.eclipse.smarthome.binding.fsinternetradio.test/META-INF/MANIFEST.MF

```diff
@@ -10,9 +10,9 @@ Import-Package: javax.servlet,
  javax.servlet.http,
  org.apache.commons.io,
  org.eclipse.jdt.annotation;resolution:=optional,
- org.eclipse.jetty.server;version="[9.3.15,9.4.0)",
- org.eclipse.jetty.server.handler;version="[9.3.15,9.4.0)",
- org.eclipse.jetty.servlet;version="[9.3.15,9.4.0)",
+ org.eclipse.jetty.server,

```

Comment:
This has been already discusses and also this one has been added (at the time of writing) by intention (and IIRC the discussion if this has to be removed has not been finished).